.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ defaults: &defaults
     PACKER_VERSION: NONE
     GOLANG_VERSION: 1.11.2
     KUBECONFIG: /home/circleci/.kube/config
-    HELM_VERSION: v2.11.0  # Same as helm provider
+    HELM_VERSION: v2.14.0  # Same as helm provider
```

Comment:
Update the comment since this is no longer true.

Reply:
Where/what is the helm provider exactly? If the helm client and server versions are not the same then it might not work.

Reply:
> Conversation in slack:
> The answer to your question about the provider is that it is referring to the terraform helm provider. The server is called Tiller, and the client is the helm client, so provider implies the terraform thing, but I just realized that is only obvious if you’ve been doing terraform. Can be very confusing to newcomers! Sorry about that.
> 
> Yori   [27 minutes ago]
> To expand a bit, the provider embeds helm and was embedding v2.11, so we were using that for consistency because we viewed a common use case to be driving kubergrunt from terraform
> 
> Ben Whaley   [22 minutes ago]
> Ok, so if someone were to use terraform to provision a Helm chart to tiller, would terraform use helm v2.11 to do so? If so, I think that will not work if tiller v2.14 is deployed by kubergrunt as the versions would mismatch.
> 
> Yori   [18 minutes ago]
> Right, but you can tell kubergrunt to deploy v2.11 if you were using terraform by setting the `—tiller-version` flag. Also, we switched recently to a different, more terraform friendly approach for deploying Tiller (we still use kubergrunt for TLS management), so it is less relevant now than when we were directly calling deploy.

Reply:
Thanks for migrating that over!